src/commands/defi/withdraw.ts

```diff
@@ -37,6 +37,19 @@ async function getDestinationAddress(
 async function withdraw(msg: Message, args: string[]) {
   const payload = await Defi.getTransferPayload(msg, args)
   const data = await Defi.discordWalletWithdraw(JSON.stringify(payload))
+  if (data.error.includes("balance is not enough")) {
```

Comment:
should create `InsufficientBalanceError` and throw that if error message includes "balance is not enough"
check `DiscordWalletTransferError.ts` for reference